packages/lib-utils/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useEventListener } from './useEventListener';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const tryJSONParse = <T = any>(data: string): string | T => {
+  try {
+    return JSON.parse(data);
+  } catch {
+    return data;
```

Comment:
This seems a bit odd that we would return the original string value we were trying to parse without any indication in the hook that something went wrong trying to parse the localStorage. I'd be curious how the gettingStartedCard would behave in that scenario. Would it continue to re-render every page refresh since we cannot correctly parse locally storage to indicate it was dismissed already? 

 For now this is ok, but we may need to revisit it - perhaps in the component here: https://github.com/openshift/dynamic-plugin-sdk/pull/163/files#diff-de59ca7b3bb227b3f0fe8cb00f30a4e840c36509c06bc9a49d7761ab250160ebR42 to handle if the parse failed.

Reply:
I'll have to defer to @rottencandy as he's the actual author of this code.

Reply:
I was originally hoping that we'd want to be able to store both objects and strings in local storage. But perhaps it makes sense to restrict it to one type of data.

Reply:
But yeah in this case the card would render again on refresh if the local storage lookup fails.

Reply:
Gotcha @rottencandy. I think since hac-dev is already behaving in this manner we can merge this and open a follow up ticket to revisit the `tryJSONParse` behavior when parsing JSON fails (and how the card responds to it).

Reply:
https://issues.redhat.com/browse/HAC-2285